### PR TITLE
Complete the shift to the "jobs:" key for matrix

### PR DESCRIPTION
**Pull request by cclauss**
__Matrix__ is the concept but the yaml key should be __jobs:__.

Also, update some Python version numbers.

### CODE REVIEW

**Comment by cclauss** [comment]
@MariadeAnton

File: user/build-matrix.md [hunk 1]
```diff
@@ -21,7 +21,7 @@ There are two ways to specify multiple parallel jobs (what we call the build mat
   ```
   {: data-file=".travis.yml"}
 
-* specify the exact combination of configurations you want in `matrix.include`. For example, if not all of those combinations are interesting, you can specify just the combinations you want:
+* specify the exact combination of configurations you want in `job.include`. For example, if not all of those combinations are interesting, you can specify just the combinations you want:
```
Comment (MariadeAnton) [inline user/build-matrix.md]:
```suggestion
* specify the exact combination of configurations you want in `jobs.include`. For example, if not all of those combinations are interesting, you can specify just the combinations you want:
```

Comment (cclauss) [reply user/build-matrix.md]: `jobs` (plural) matches the yml syntax two lines below.

Comment (Pezi777) [reply user/build-matrix.md]: @cclauss could you please 'commit Maria's suggestion' - then I can merge the PR :-)

**Pull request merged**